.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: cpp
-dist: trusty
+dist: bionic
```

Comment:
I'm still cautious about changes to the minimum compiler version.

Did this turn out to still be necessary?

Reply:
Maybe I can move back to trusty. Would you want to do that?

Reply:
If it's practical.

If trusty is really too old, and there's not an "easy" solution to the compiler issues, then I would prefer to keep supporting trusty, but I'm not opposed to Bionic, as long as the compiler series is still the GCC 7.x.y release.

Like I said, I'm currently stuck on 7.3.0, which supports most of C++17, and it looks like Bionic has GCC 7.4.0, which is probably close enough.

But I'm saying that if there isn't otherwise a reason to switch to the newer version (because compiler errors, for example) then we shouldn't change the minimum version.

Reply:
Ok, I will downgrade to trusty.

Reply:
I just confirmed the current version of impl_320_2 works fine with trusty. I will use trusty.

Reply:
done(just my memo, not pushed yet).

Reply:
This change does not appear to have been reverted.

Did you revert it?